tests/scanner/audit/data/bucket_retention_test_rules_21.yaml

```diff
@@ -0,0 +1,40 @@
+rules:
+  - name: retention of all buckets is 365 days
+    applies_to:
+      - bucket
+    resource:
+      - type: bucket
+        resource_ids:
+          - some-resource-id1
+    minimum_retention: 365
+    maximum_retention: 365
+  - name: retention of buckets in all projects must be 365 days
```

Comment:
the max_retention is 2147483647 not 365, am i missing something?

Reply:
Yes. the maximum value of the age is the max value of Integer but this is not that meaning. This is a rule instead of a config.